tests/acceptance/survey/2016-test.js

```diff
@@ -28,7 +28,10 @@ module('Acceptance | survey/2016', function (hooks) {
   test('Accessibility audit', async function (assert) {
     await visit('/survey/2016');
     await waitUntilAllChartsAreDrawn();
-    await a11yAudit();
+    await a11yAudit({
+      exclude: [['[data-test-chart]']],
+      include: [['#ember-testing-container']],
+    });
```

Comment:
We'll assume that Highcharts does create an accessible chart. The svg element, which contains many elements, can slow down an application test (by an order of 0.1s - 10s).